### DISCUSSION TITLE
NPM script words with f all have spaces.

### DISCUSSION THREAD

**Discussion by regularguy01** [tteck/Proxmox#2373] [General]
Hey  wanted you to know the nginx proxy manger script.  Is adding a 'space'  between words with f.  example f or, def ault, and even `/bin` files like `f allocate`.
a few screen shots
![image](https://github.com/tteck/Proxmox/assets/22330436/1285d126-cadc-49f0-ba1f-ab26b0c125c7)
![image](https://github.com/tteck/Proxmox/assets/22330436/49e7a030-23b6-47b0-a22c-2a8cdabdbbbe)
![image](https://github.com/tteck/Proxmox/assets/22330436/076a2d42-8ec6-4694-9fcb-c9de5bd14055)
![image](https://github.com/tteck/Proxmox/assets/22330436/7268c47d-11cc-4bd3-a165-e9d861829f63)

I havent tried others yet but wanted you to know.

**Comment by tteck** [comment]
Not on my end.

![Screenshot 2024-01-20 4 16 58 PM](https://github.com/tteck/Proxmox/assets/21342670/d87a6a8b-ba45-4c96-93a2-528b7a859fab)

![Screenshot 2024-01-20 4 17 47 PM](https://github.com/tteck/Proxmox/assets/21342670/2c069297-e781-469f-91ab-95800216448a)

**Comment by regularguy01** [comment]
Oh man super weird.  I do have some Proxmox updates.  What version are you on?

**Comment by regularguy01** [comment]
Its something to do with the Proxmox Console
![image](https://github.com/tteck/Proxmox/assets/22330436/191f6fa6-c08c-4a7c-afa0-a9004962ca42)

If I ssh it looks fine.
![image](https://github.com/tteck/Proxmox/assets/22330436/76bd5825-6c74-4601-92f7-d796d534f75c)

weird.

Going to update and reboot.

**Comment by regularguy01** [comment]
Its something to do with xterm.js

I am on Proxmox 8.14

**Reply by tteck** [reply]
No issues here with `xterm.js`

**Comment by regularguy01** [comment]
Ok Narrowed it down.
I am on ArcoLinux with KDE.
Chromium = No render issue
Firefox = No render issue
Librewolf = Has issue.  What I am using as Browser.  Which is why I saw the issue.

So in Summary
Not a script issue
Not a xterm.js issue
Browser Librewolf font issue.
I have a Mac to but haven checked that yet.
